native-assets/fractional-NFT/project/SPECIFICATION.md

```diff
@@ -0,0 +1,91 @@
+Table of Contents
+- [Overview](#overview)
+- [Use Cases](#use-cases)
+- [Fractional NFT Contract](#fractional-nft-contract)
+  - [SRC-6](#src-6)
+    - [`deposit()`](#deposit)
+    - [`withdraw()`](#withdraw)
+    - [`managed_assets()`](#managed_assets)
+    - [`max_depositable()`](#max_depositable)
+    - [`max_withdrawable()`](#max_withdrawable)
+  - [SRC-20 Functionality](#src-20-functionality)
+    - [`total_assets()`](#total_assets)
+    - [`total_supply()`](#total_supply)
+    - [`name()`](#name)
+    - [`symbol()`](#symbol)
+    - [`decimals()`](#decimals)
+    - [`set_name()`](#set_name)
+    - [`set_symbol()`](#set_symbol)
+
+# Overview
+
+This document provides an overview of the application.
+
+It outlines the use cases, i.e. desirable functionality, in addition to requirements for the smart contract and the user interface.
+
+# Use Cases
+
+This section contains general information about the functionality of the application and thus does not touch upon any technical aspects.
+
+If you are interested in a functional overview then this is the section for you.
+
+# Fractional NFT Contract
+
+## SRC-6
+
+### `deposit()`
+
+This function will mint new shares when an NFT is deposited.
+
+### `withdraw()`
+
+This function will burn shares and release the locked NFT.
```

Comment:
In my anticipation of the implementation I propose that this should be changed to outline that once all shares are returned (by all parties?) then the NFT will be released